.github/workflows/publish-canary.yml

```diff
@@ -4,6 +4,7 @@ on:
   push:
     branches:
       - develop
+      - release
```

Comment:
Need to handle dates in the release name

```suggestion
      - release-*
```

Reply:
(discussed on slack)